systems-manager.tf

```diff
@@ -0,0 +1,96 @@
+# AWS KMS alias used for encryption/decryption of SSM secure strings
+variable "kms_alias_name_ssm" {
```

Comment:
Why put the `variable` and `output` block into this file? I get that they're isolated to systems manager related functionality, but I'd still prefer to see them all live in their typical places (i.e. `variables.tf` and `outputs.tf`).

Reply:
I think because this was a precursor to putting it in its own module eventually. That way it would be easier to migrate. We can put it into `variables.tf`, it would just make migration slightly more difficult.

We frequently do this in our components. Have a specific `<service>-variables.tf` or `variables-<service>.tf` file so it's easier for copy pasta. I figured the same methodology could be applied in the modules too.